distance

```diff
@@ -1,3 +1,10 @@
+def dist x,y
```

Comment:
Make a point class; take two point objects as arguments to your distance method.